sphinx-docs/screenshot.py

```
@@ -23,19 +23,41 @@ def _parse_focus(arg_str):
         return {'id': split_str[0], 'annotation': split_str[1]}
 
 def _parse_command(command):
-    # Return a dictionary with
-    #   action: the action type (if it's recognized)
-    #   options: a list of options
-    raise NotImplementedError()
+    """" Parses a command into action and options.
+
+    Returns a dictionary with following keys:
+       action (string): the action type (if it's recognized)
+       options (list): a list of options
+
+       Both keys have empty values if action type is not recognized.
```

If the action type is not recognized, let's raise an error.